src/example-relay/src/components/Navbar.js

```diff
@@ -171,6 +185,10 @@ const styles = sx.create({
       outline: 'none',
     },
   },
+  brand: {
+    animation: `${brandAnimation} 1s`,
```

Comment:
Such string concatenation will make the static analysis quite a bit more challenging when building a Babel plugin for SX. I guess (in this case) it's needed especially because of the shorthand property. Have you considered disallowing this shorthand property OR allowing keyframes only in `animation-name`? Note that shorthand properties are explicitly discouraged in our readme.

Reply:
> Note that shorthand properties are explicitly discouraged in our readme

Yeah, I know, I just think people will use them anyway because it is convenient 🙈